src/components/molecules/ButtonHome.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import IconButton from 'src/components/atoms/IconButton';
+import Subtitle from 'src/components/atoms/SubTitle';
+import Stack from '@mui/material/Stack';
+
+type ButtonHomeTypes= {
```

Comment:
Why are we using "type" instead of "interface" for the props?

Reply:
I made some changes, could you tell me if the way of using the file types is correct, please? Consequently, I'll continue with the other files in the same way.